docs/en/changes/changes.md

```diff
@@ -3,6 +3,7 @@
 #### Project
 
 * Bump up the embedded `swctl` version in OAP Docker image.
+* Migrate to BanyanDB v0.2.0.
```

Comment:
I think this should be in `backend` only?

Reply:
Also, please make this a breaking change. BanyanDB server breaks the protocol, and we change the ways to store data.

Reply:
Added